bin/run.sh

```diff
@@ -112,6 +86,12 @@ get_test_bodies() {
                 if [[ $line =~ $end_test_re ]]; then
                     test_bodies["$name"]=$(printf '%s\n' "${body[@]}")
                     state="out"
+                elif [[ $line == *BATS_RUN_SKIPPED*skip* ]]; then
+                    # skip the skips
+                    continue
+                elif ((${#body[@]} == 0)) && [[ $line == *([[:blank:]]) ]]; then
+                    # ignore blank lines at the top of the test body
+                    continue
```

Comment:
This is the important new code.

Reply:
I'm on my phone so ... do you explicitly set extglob in this script?

Reply:
extglob is enabled automatically inside `[[...]]`